src/Makefile.test.include

```diff
@@ -224,11 +224,11 @@ test_test_bitcoin_LDADD += $(LIBBITCOIN_ZMQ) $(ZMQ_LIBS)
 endif
 
 if ENABLE_FUZZ
-test_fuzz_block_deserialize_SOURCES = $(FUZZ_SUITE) test/fuzz/deserialize.cpp
-test_fuzz_block_deserialize_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES) -DBLOCK_DESERIALIZE=1
```

Comment:
You removed the compilation of this target, but not the code. Maybe it is better to leave the target and compile it for now.

Reply:
Oh, good catch! Thanks! Now leaving the target instead.